code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -81,7 +81,8 @@ emp_act
 //End Here
 
 /mob/living/carbon/human/proc/check_shields(damage = 0, attack_text = "the attack", atom/movable/AM, thrown_proj = 0, armour_penetration = 0)
-	var/block_chance = 50 + 30*thrown_proj - round(damage / 3) //thrown things are easier to block
+	var/block_chance = 50 + (thrown_proj ? 30 : 0) - round(damage / 3) //thrown things are easier to block
+	block_chance = max(block_chance - armour_penetration, 0)

```

Comment:
this doesn't fix the origin of the runtime, it just hides it. It runtimes because someone called check_shields with a string as 4th argument. (caused by https://github.com/tgstation/-tg-station/commit/f1e12a27831808b69822890d096f69effd4b585d)